virt_lightning/__init__.py

```diff
@@ -54,16 +54,16 @@ def __init__(self, configuration, uri="qemu:///session"):
         )
         self.get_storage_dir()
         self.wait_for = []
+        self.kvm_binary = self.find_kvm_binary()
 
     def create_domain(self):
-        domain = LibvirtDomain.new(self)
-        domain.cloud_init = {
-            "resize_rootfs": True,
-            "disable_root": 0,
-            "bootcmd": [],
-            "runcmd": [],
-        }
-        return domain
+        root = ET.fromstring(DOMAIN_XML)
+        e = root.find("./name")
+        e.text = str(uuid.uuid4())[0:10]
```

Comment:
Use `.hex` instead converting to string via `str()`.
```
uuid.uuid4().hex[0:10]
```

Reply:
:+1: